templates/2_create_tiled_mibi_run.ipynb

```diff
@@ -13,7 +13,9 @@
    "cell_type": "markdown",
```

Comment:
<p>What's the point of having the user create another directory? We should link to the readme explaining the directory structure. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
This was requested by @HPiyadasa, who wanted separate directories inside `tiled_regions_json` as a way to organize the the various region corners and slide files. This is similar to the problem we experienced during co-registration (addressed by #196), where users would put their copied notebook files into separate subdirectories inside of `templates` to eliminate clutter.

@ngreenwald I'll remove this for the time being. @HPiyadasa we can revisit this issue at a later time, but for the MIBI workshop it's might be easier to just use the main `tiled_regions_json` folder.

Reply:
@HPiyadasa what was the issue that having a sub-folder is addressing?